### ISSUE TITLE
Spawning of some mobs is broken

### ISSUE THREAD

**Issue by WitheredWasabi** [8s2/Angling#3]
At the very least Crabs, Anglers, Anomalocaris, and Nautilus don't seem to spawn at all in regular worlds.

I did some extra testing to see if Single Biome worlds worked with Crabs, and they seemed to spawn fine in them. I'm using the latest versions of everything without any extra mods.

**Comment by TamaT1ME** [comment]
Yeah, I'm doing that too, what I found is most of these creatures don't seem to spawn naturally as far as I've experienced. As well, I saw that the sea slug eggs spawning and texture/model is broken for me for some reason.

**Comment by UndoZan** [comment]
Is this beeing worked on?
